src/lib/Libifl/pbs_loadconf.c

```diff
@@ -1034,11 +1027,10 @@ __pbs_loadconf(int reload)
 		}
 	}
 
-	if (pbs_conf.pbs_use_tcp == 0) {
-		pbs_conf.pbs_use_compression = 0;
-		pbs_conf.pbs_use_mcast = 0;
-		pbs_conf.pbs_use_ft = 0;
-	}
+	set_log_conf(pbs_conf.pbs_leaf_name, pbs_conf.pbs_mom_node_name, pbs_conf.locallog,
```

Comment:
pbs_loadconf dependence on liblog may not be a very good thing

Reply:
Hmm, I didn't think much on that part earlier, but you are right, so I have moved `set_log_conf()` outside of `pbs_loadconf()` and added at appropriate places in code where we actually using liblog library.